mne/io/egi/events.py

```diff
@@ -60,6 +72,12 @@ def _read_mff_events(filename, sfreq, nsamples):
         for event in orig[xml][2:]:
             event_start = _ns2py_time(event['beginTime'])
             start = (event_start - start_time).total_seconds()
+            # adjust the start to account for epochs
+            epoch = [i[0] for i in enumerate(epoch_times)
```

Comment:
it would be a bit easier to read this way:
```python
epoch_idx = [idx for idx, times in enumerate(epoch_times)
             if (times[0] <= start and times[1] >= start)]
```